### DISCUSSION TITLE
"Getting started" steps dont count as done when done before other steps

### DISCUSSION THREAD

**Discussion by hasezoey** [voxeltycoon/issues#594] [General]
*just started the steam version (with "getting started" enabled)*
multiple steps dont count as done when done before being asked for
like step doing step 4,5,6,7,8,9 before step 3 ("build a mine on deposit")

reproduction:
1. build street to deposit
2. build garage on street
3. build station near deposit (on street)
4. build station near customer (on street)
5. buy cars
6. setup schedules
7. build mine
8. "getting started" finished step 3 ("build a mine") but not step 4 ("build a station near the mine")

**Comment by voxeltycoon** [comment]
Related: https://github.com/voxeltycoon/issues/discussions/600
